resources/md/guestbook.md

```diff
@@ -416,8 +416,8 @@ denoted using `:` notation. Let's replace the existing queries with some of our
 -- :name save-message! :! :n
 -- :doc creates a new message
 INSERT INTO guestbook
-(name, message)
-VALUES (:name, :message)
+(name, message, timestamp)
```

Comment:
without this, the example call from REPL (below) did not set timestamp properly:
```
(def query-fn (:db.sql/query-fn state/system))

(query-fn :save-message! {:name      "m1"
                          :message   "hello world"
                          :timestamp (java.util.Date.)})
```